napari/_qt/_tests/test_qt_viewer.py

```diff
@@ -675,3 +679,123 @@ def test_create_non_empty_viewer_model(qtbot):
     del viewer
     qtbot.wait(50)
     gc.collect()
+
+
+# The following are integration tests for the new style of slicing.
+# They are marked with sync_only because that denotes that the old experimental
+# async should not run as we don't explicitly wait for its threads to finish.
```

Comment:
I'm not actually sure if the old experimental async will work when `LayerSlicer._force_sync` is `False`. I don't think it's too important for these tests as they are supposed to integration tests for the new async, but maybe that's more reason to move these somewhere else.